modules/ui.py

```diff
@@ -1229,11 +1231,21 @@ def refresh():
                     with gr.Row():
                         train_hypernetwork_name = gr.Dropdown(label='Hypernetwork', choices=[x for x in shared.hypernetworks.keys()])
                         create_refresh_button(train_hypernetwork_name, shared.reload_hypernetworks, lambda: {"choices": sorted([x for x in shared.hypernetworks.keys()])}, "refresh_train_hypernetwork_name")
+
+                    # support APT
+                    gr.HTML(value='<p style="margin-bottom: 0.7em">May be batter used with "Stop At last layers of CLIP model = 1"</p>')

```

Comment:
To be honest, this feels out of place. This should be a tooltip (in the file javascript/hints.js) or added later to the wiki.